src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/ProcessorTestData.cs

```diff
@@ -0,0 +1,19 @@
+using System.Reflection;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests;
+
+public record ProcessorTestData(string Filename)
+{
+    private static string DataFilesDirectoryPath => Path.Combine(
+        Assembly.GetExecutingAssembly().GetDirectoryPath(),
+        "Resources",
+        "DataFiles"
+    );
+
+    public static ProcessorTestData AbsenceByCharacteristic2016 => new(nameof(AbsenceByCharacteristic2016));
```

Comment:
Just remembered that these files aren't Absence By Characteristic 2016 at all, they are a copy of the test files taken from the Public API, so I will need to rename this to something more suitable. I might also revert the structure back to using an additional folder to group the files by as it would better support other test data sets being added to this project that might have several files like the set of parquet files.

Reply:
Yep! I was about to leave you a comment saying exactly this :smile:

Reply:
🙂 I've reverted the move of those test data files now and made some changes to `ProcessorTestData`. Changes are in the first fixup commit.